simulariumio/file_converter.py

```diff
@@ -19,24 +21,207 @@
 class FileConverter(TrajectoryConverter):
     def __init__(self, input_file: InputFileData):
         """
-        This object loads the data in .simularium JSON format
-        at the input file
+        This object loads data from the input file in .simularium format.
 
         Parameters
         ----------
         input_file: InputFileData
-            A InputFileData object containing a string path
-            or string contents for the .simularium JSON file to load
+            A InputFileData object containing .simularium data to load
         """
-        print("Reading Simularium JSON -------------")
-        buffer_data = json.loads(input_file.get_contents())
+        try:
+            print("Reading Simularium JSON -------------")
+            buffer_data = json.loads(input_file.get_contents())
+        except UnicodeDecodeError:
+            print("Reading Simularium binary -------------")
+            buffer_data = FileConverter._load_binary(input_file)
```

Comment:
update to allow `FileConverter` to read in binary files

Reply:
The above try/except would probably be a lot more efficient if you could just read the first 16 bytes of the file and compare them with "SIMULARIUMBINARY" with a regular old if statement.  That check could be encapsulated as a little function in FileConverter, like `if is_binary_file(input_file):`

Reply:
is it `get_contents` that raises the UnicodeDecodeError ?

Reply:
added `InputFileData._is_binary_file()` which reads the header to check for binary